cf_remote/commands.py

```diff
@@ -935,13 +935,20 @@ def agent(hosts, bootstrap=None):
 
 
 def connect_cmd(hosts):
-
-    assert hosts and len(hosts)
+    assert hosts and len(hosts) >= 1  # Ensured by argument parser
 
     if len(hosts) > 1:
         user_error("You can only connect to one host at a time")
 
     print("Opening a SSH command shell...")
-    subprocess.run(["ssh", hosts[0]])
-
-    return 0
+    r = subprocess.run(["ssh", hosts[0]])
+    if r.returncode == 0:
+        return 0
+    if r.returncode < 0:
+        print("")
+        log.error("The ssh command failed with signal " + str(abs(r.returncode)))
```

Comment:
`print()` writes to `stdout` and `log.error()` probably writes to `stderr`. Was this intended?

Reply:
No, all the log functions log to stdout:

https://github.com/cfengine/cf-remote/blob/84a5234088c6a0cb5aee836e11fc866f3642e057/cf_remote/log.py#L17